### DISCUSSION TITLE
Can I run a script on network change?

### DISCUSSION THREAD

**Discussion by aikooo7** [termux/termux-app#4065] [Q&A] [answered]
Hello everyone, I have some specific stuff that I need to run on network change, is it possible?

**Comment by ByteJoseph** [comment]
Do you mean to detect when the connection switches between mobile data and Wi-Fi?

**Comment by agnostic-apollo** [comment, marked as answer]
Android events are not supported by Termux, you use Tasker app or other automation apps along with Termux:Tasker plugin for that.
